main-app/app/scripts/providers/model/n-and-c-model.js

```diff
@@ -0,0 +1,165 @@
+(function () {
+    'use strict';
+    angular.module("MainApp")
+        .factory("NAndCModel", ['$http', '$location', function($http, $location) {
+
+            var NAndCModel = function() {
+
+                var me = this;
+                this.currentPlayer = 0;
+                var p1State = 'human';
+                var p2State = 'human';
+
+                this.gameActive = false;
+
+                this.board = [];
+
+                this.startNewGame = function() {
+
+                    $http.post("http://noughtsandcrosses:35000/api/v1.0/newgame", {player1: p1State, player2: p2State}, {withCredentials: true})
```

Comment:
Don't fix for this PR - but you should inject the $http functionality as a separate provider proxy